tools/validate_configs/test_configs/config-ssh.yaml

```diff
@@ -46,54 +46,53 @@ deployment_groups:
       configure_ssh_host_patterns: ['10.182.0.*', 'hpcsm*']
 
   - id: debug_node_group
-    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network1]
     settings:
       node_count_dynamic_max: 4
       machine_type: n2-standard-2
+      enable_placement: false # the default is: true
 
   - id: debug_partition
-    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    source: community/modules/compute/schedmd-slurm-gcp-v6-partition
     use:
-    - network1
     - homefs
     - debug_node_group
     settings:
       partition_name: debug
       exclusive: false # allows nodes to stay up after jobs are done
-      enable_placement: false # the default is: true
       is_default: true
 
   - id: compute_node_group
-    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network1]
```

Comment:
Lets update to `network` in the blueprint.

Reply:
I replaced `network1` with `network`